server/submission/queries.ts

```diff
@@ -0,0 +1,34 @@
+import { Submission } from 'server/models';
+
+export const createSubmission = async (inputValues, actorId = null) =>
+	Submission.create(
+		{
+			pubId: inputValues.pubId,
+			status: inputValues.status,
+		},
+		{ actorId },
+	);
+
+export const updateSubmission = async (inputValues, updatePermissions, actorId = null) => {
+	const filteredValues: Record<string, any> = {};
```

Comment:
This whole thing could be replaced by `lodash.pick`!

Reply:
This was copied over as boilerplate, seems pretty much unnecessary with a model that has basically one editable field.